i3-renameworkspaces.pl

```diff
@@ -86,6 +87,25 @@ sub updatelabels {
     });
 }
 
+sub locate_config_file {
+    my $filename = "i3workspaceconfig";
+    my $filepath = "";
+
+    if ( defined( $ENV{'XDG_CONFIG_HOME'} ) ) {
```

Comment:
Could you create a list of possible candidate locations, and then loop through them, taking the first one that exists and eventually fall back to i3workspaceconfig?